NuRadioReco/examples/cr_efficiency_analysis/1a_threshold_estimate_fast.py

```diff
@@ -0,0 +1,306 @@
+import scipy.constants
```

Comment:
yes, that would be nice. I already tried this with the parameter trigger name. But since the script 2a is made to run 2e6 times, i thought it is the best to keep it as easy as possible.

Reply:
You can just define a helper function that the two scripts access.